tutor/src/screens/task/external.js

```diff
@@ -1,46 +1,15 @@
-import {
-  React, PropTypes, observer, computed, action,
-} from 'vendor';
-import ContentLoader from 'react-content-loader';
+import { React, PropTypes, observer } from 'vendor';
 import UX from './ux';
 import Instructions from './step/instructions';
 import withFooter from './with-footer';
 
-const Loader = () => (
-  <ContentLoader>
-    <rect x="0" y="0" rx="3" ry="3" width="250" height="10" />
-    <rect x="0" y="10" rx="3" ry="3" width="250" height="10" />
-  </ContentLoader>
-);
-
 @observer
 class ExternalTaskStep extends React.Component {
 
-  static Loader = Loader;
-
   static propTypes = {
     ux: PropTypes.instanceOf(UX).isRequired,
   }
 
-  onContextMenu(ev) {
-    return ev.preventDefault();
-  }
-
-  @computed get url() {
-    const { ux: { task } } = this.props;
-    let { external_url } = task.steps[0];
-    if (!/^https?:\/\//.test(external_url)) {
-      external_url = `http://${external_url}`;
-    }
-    return external_url;
-  }
-
-  @action.bound onContinue() {
-    const step = this.props.ux.currentStep;
-    step.is_completed = true;
-    step.save();
-  }
```

Comment:
I see that `onContextMenu` isn't being used but that is a bug and we need to hook it back up.  

Right clicking on the link isn't allowed since the only valid use case is opening in a new tab and we already do that for them.  We need to record the visit and we can't do that if they right-click and open in new tab.

Reply:
good to know. I will put it back.